components/ErrorDisplay/ErrorDisplay.js

```diff
@@ -21,11 +21,13 @@ function ErrorDisplay({ statusCode }) {
         <meta name="robots" content="noindex, nofollow" key={statusCode} />
       </Head>
 
-      <div className={styles.ErrorDisplay}>
-        <div className={styles.bg}>
-          <div className={styles.textContainer}>
-            <h1 className={styles.title}>{statusCode || 'Oh no'}!</h1>
-            <p className={styles.paragraph}>
+      <div className={`${styles.ErrorDisplay} bg-[#121212] w-[100% h-[100%]`}>
```

Comment:
We don't need to import or use `styles` anymore!

Reply:
then shall I remove the CSS file too?